cmd/vic-init/ops.go

```diff
@@ -42,7 +42,12 @@ type operations struct {
 	tether.BaseOperations
 }
 
-func (t *operations) Setup() error {
+func (t *operations) Setup(sink tether.ConfigSink) error {
+	if err := t.BaseOperations.Setup(sink); err != nil {
+		return err
+	}
+
+	// TODO: enabled for initial dev debugging only
```

Comment:
No longer just for dev debugging - now it's restricted to loopback it's there for vicadmin to gather as part of diagnostic logs.